docs/user-guide/getting-started.md

```diff
@@ -37,12 +37,14 @@ After that, you can run ESLint in your project's root directory like this:
 $ ./node_modules/.bin/eslint yourfile.js
 ```
 
-Note that you may also use `npx` to run `eslint`:
+Instead of navigating to `./node_modules/.bin/` you may also use `npx` to run `eslint`:
 
 ```
 $ npx eslint
 ```
 
+**Note:** If wasn't manually installed (via `npm`), `npx` will install `eslint` to a temporary directory and execute it.
```

Comment:
Suggestion: "If ESLint wasn't manually installed"

@g-plane Can you comment on the accuracy of "a temporary directory"? Is it really a temporary directory or does not install to the global installation location? Thanks!